include/oidcc_token_introspection.hrl

```diff
@@ -3,9 +3,17 @@
 -record(oidcc_token_introspection, {
```

Comment:
I assume you based this on this RFC: https://datatracker.ietf.org/doc/html/rfc7662#section-2.2

Can you add a comment with the link to it? (Both to the record and the struct.)